.github/workflows/keg-format.yml

```diff
@@ -0,0 +1,39 @@
+
+name: keg-format
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  tests:
+    name: 'Format Repo Code'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: "Get Changed Files"
+        id: get-changed-files
+        run: bash scripts/ci/getChangedFiles.sh
+
+      - name: "Get Changed Repos"
+        id: get-changed-repos
+        run: bash scripts/ci/getChangedRepos.sh
+
+      - name: "Install Yarn Dependencies"
+        id: install-yarn-dependencies
+        run: bash scripts/ci/runYarnCmd.sh "install"
+
+      - name: "Run Repo Format"
```

Comment:
* It **should** not be possible for the linter to break our code. 
* Keyword is should, cause anything is possible, the whole point of a linter is to enforce a correct coding standard.
  * I've never heard of a linter that formats code in a way that breaks it. What would be the point?
* So yes, it's possible but pretty sure most the internet would break as well.
  * Prettier / ESLint are _almost_ a requirement these days for most production grade repositories.

Honestly, I think it's a non-issue

Reply:
i figured it would be a bit of a stretch. works for me